core/domain/protofiles/text_classifier.proto

```diff
@@ -0,0 +1,21 @@
+// coding: utf-8
```

Comment:
Regarding the directory structure, I think it's typical to include protos under a 'proto' directory rather than 'protofiles', but that may be different for Python projects if you've seen any evidence to the contrary.

Reply:
I think we can replace `protofiles` with `proto`. I don't see any issues with that.

Reply:
Renamed.